### PR TITLE
🔧 Accessibilty revisions

### PR DESCRIPTION
**Pull request by davidreneuw**
# Pull Request

## Description

Implemented a few changes to reflect the comments left on my previous accessibility PR

## Changes

- Moved accessibility css to its own dedicated folder/files
- Removed wet-boew theme as it is currently unnecessary
- Implemented better styling for DHButtons instead of splattering inline styles everywhere
- Fixed "Class" typo in MudItem

## Testing

- All unit tests passed, no new tests

## Checklist

- [x] Code follows dotnet coding standards
- [x] Tests added/updated to cover changes

### CODE REVIEW

File: Portal/src/Datahub.Portal/Components/Announcements/AnnouncementCarousel.razor [hunk 1]
```diff
@@ -23,7 +23,7 @@
                                           Color="@Color.Primary"
                                           Variant="@Variant.Text"
                                           EndIcon="@Icons.Material.Filled.KeyboardDoubleArrowRight"
-                                          Style="text-wrap: nowrap; text-decoration: underline">
+                                          Underline NoWrap>
```
Comment (yjmrobert) [inline Portal/src/Datahub.Portal/Components/Announcements/AnnouncementCarousel.razor]:
```suggestion
                                          Underline 
                                          NoWrap>
```

**Review by yjmrobert — DISMISSED** [review]
Awesome!

**Review by Sean-Stilwell — DISMISSED** [review]
Tested locally, everything working as expected

LGTM!

**Review by Sean-Stilwell — APPROVED** [review]
Re-approving the verified commits, LGTM still!

**Pull request merged**